src/config/config.CSSStub.js

```diff
@@ -0,0 +1 @@
+module.exports = {}
```

Comment:
whats this

Reply:
This is for the jest tests. Its a stub file for jest to read, because it cannot read the style imports, so it's stub as though its a component